sencha-workspace/SlateTasksManager/app/controller/Tasks.js

```diff
@@ -155,6 +159,13 @@ Ext.define('SlateTasksManager.controller.Tasks', {
         tasksStore.load();
     },
 
+    onIncludeUnsharedCheckboxClick: function(checkbox) {
+        var tasksStore = this.getTasksStore();
+
+        tasksStore.getProxy().extraParams.include_unshared = checkbox.checked;
```

Comment:
Better to use the API method:

```suggestion
        tasksStore.getProxy().setExtraParam('include_unshared', checkbox.checked);
```

Reply:
resolved: [9bd19ea](https://github.com/SlateFoundation/slate-cbl/pull/710/commits/9bd19ea543cca4730c594bcee0f294406ffa6b6c)